src/components/casestudyandcoretools/Casestudy.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Entries, Entry } from "@/types/interfaces";
+import InformationCard from "../cards/InformationCard";
+import { ArcherContainer } from "react-archer";
+
+interface Props {
+  categorizedEntries: { level1: Entries; level2: Entries; level3: Entries };
+}
+
+const Casestudy = ({ categorizedEntries }: Props) => {
+  return (
+    <div>
+      <ArcherContainer>
```

Comment:
Remove this Archer container as there will be no arrow in this page

Reply:
Removed